components/admin_console/billing/billing_subscriptions/index.tsx

```diff
@@ -63,11 +63,36 @@ const BillingSubscriptions: React.FC<Props> = () => {
 
     const [showCreditCardBanner, setShowCreditCardBanner] = useState(true);
 
+    const product = useSelector((state: GlobalState) => {
+        if (state.entities.cloud.products && subscription) {
+            return state.entities.cloud.products[subscription?.product_id];
+        }
+        return undefined;
+    });
+
+    let subscriptionPlan = 'CLOUD_PROFESSIONAL';
+
+    switch (product?.name) {
+    case 'Cloud Starter':
+        subscriptionPlan = 'CLOUD_STARTER';
+        break;
+    case 'Cloud Professional':
+        subscriptionPlan = 'CLOUD_PROFESSIONAL';
+        break;
+    case 'Cloud Enterprise':
+        subscriptionPlan = 'CLOUD_ENTERPRISE';
+        break;
+    default:
+        subscriptionPlan = 'CLOUD_PROFESSIONAL';
+        break;
+    }
+
     let isFreeTrial = false;
     let daysLeftOnTrial = 0;
     if (subscription?.is_free_trial === 'true') {
         isFreeTrial = true;
         daysLeftOnTrial = getRemainingDaysFromFutureTimestamp(subscription.trial_end_at);
+        subscriptionPlan = 'FREE_TRIAL';
```

Comment:
We shouldn't change the type of plan if they are in the free trial, in the future, they may be in a trial for enterprise or a trial for professional. We should based any trial decisions in the `isFreeTrial` bool above